data/json/mutations.json

```diff
@@ -3897,6 +3897,16 @@
         "prereqs" : ["ROT2"],
         "category": ["MUTCAT_ALPHA"],
         "healing_awake" : -0.0008
+    },{
+        "type" : "mutation",
+        "id" : "NO_HEALING_RESTING",
+        "name" : "Lost resting healing",
+        "points" : -2,
```

Comment:
Points should probably be much more than 2 - this mutation would make the game way harder.

Reply:
As I see this is more a ballance issue, because in the midgame characters have a lot of unused medicine. Even in the beginning of the game I can sleep till the morning instead of using first aid kit. So I made this mutation for my personal tastes. Should it have more variations, like below?
- SLOWHEALER, -2 pts, -0.0025 healing (-25% healing from resting) (default)
- SLOWHEALER2, -4 pts, -0.005 healing (-50% healing from resting)
- SLOWHEALER3, -6 pts, -0.009 healing (-90% healing from resting)
- SLOWHEALER4, -8 pts, -0.01 healing (-100% healing from resting)
All of them will be availabe from the start, and not valid for random mutations.